cli.js

```diff
@@ -23,6 +24,8 @@ Options:
 	--length-prefixed  -l  Read input as length-prefixed.
 	                       See https://www.npmjs.com/package/length-prefixed-stream
 	--json  -j             Output JSON instead of a pretty represenation.
+	--ndjson  -n           Output newline-delimited JSON.
```

Comment:
This is a breaking change. The backwards-compatible way would be to add a flag e.g. `--single-json`.

Reply:
ah I must've misunderstood your previous suggestion about introducing breaking change and bumping the version,
I can also make it non-breaking and increment minor version not major then, correct?